adx/resource_adx_table_ingestion_batching_policy.go

```diff
@@ -0,0 +1,103 @@
+package adx
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+
+	"encoding/json"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+type TableIngestionBatchingPolicy struct {
+	MaximumBatchingTimeSpan string
+	MaximumNumberOfItems int
+	MaximumRawDataSizeMB int
+}
+
+func resourceADXTableIngestionBatchingPolicy() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceADXTableIngestionBatchingPolicyCreate,
+		ReadContext:   resourceADXTableIngestionBatchingPolicyRead,
+		DeleteContext: resourceADXTableIngestionBatchingPolicyDelete,
+		UpdateContext: resourceADXTableIngestionBatchingPolicyCreate,
```

Comment:
At least by azurerm convention - if we use the same func to do both Create and Update - func name is `resourceXXXCreateUpdate`. Could we amend that please?

Reply:
fixed